visual_regression_tracker/visualRegressionTracker.py

```diff
@@ -14,41 +14,49 @@ def __init__(self, config: Config):
         self.config = config
         self.headers = {'apiKey': self.config.apiKey}
 
-    def _startBuild(self):
-        if self.buildId:
-            return
+    def _isStarted(self):
+        return self.buildId is not None and self.projectId is not None
 
+    def start(self):
         data = {
             'branchName': self.config.branchName,
             'project': self.config.project,
         }
-        result = _http_post_json(
+        result = _http_request(
             f'{self.config.apiUrl}/builds',
+            'post',
             data,
             self.headers
         )
         build = _from_dict(result, Build)
-
-        if build.id:
-            self.buildId = build.id
-        else:
-            raise Exception("Build id is not defined")
-
-        if build.projectId:
-            self.projectId = build.projectId
-        else:
-            raise Exception("Project id is not defined")
+        self.buildId = build.id
+        self.projectId = build.projectId
+
+    def stop(self):
```

Comment:
Is it acceptable to submit additional test results after the build is stopped? Perhaps the stop function should clear the buildId and projectId.

Reply:
good point! 
let me change it

Reply:
updated